swap_meet/vendor.py

```diff
@@ -1,2 +1,64 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None) :
+        if not inventory:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+
+    def get_by_category(self, category):
+        item_list = []
+        for item in self.inventory:
+            if category == item.category:
+                item_list.append(item)
+
+        return item_list
```

Comment:
Looks good.

This method is a great candidate for using list comprehension if you want to refactor it since our conditional statement we check is pretty simple.

General syntax for list comp:
```python
result_list = [element for element in source_list if some_condition(element)]
```

Which here would look like:

```python
item_list = [item for item in self.inventory if item.category == category]
```

You can also forgo saving the list to the variable item_list and do something like:
```python 
def get_by_category(self, category):
    return [item for item in self.inventory if item.category == category]
```